compiler/plc_driver/src/pipelines.rs

```diff
@@ -350,8 +350,8 @@ impl<T: SourceContainer + Sync> AnnotatedProject<T> {
                                 .with_internal_error(it.into())
                             })?
                         } else if unit_location.has_root() {
-                            let root = Path::new("/").canonicalize()?;
-                            unit_location.strip_prefix(root).expect("Name has root")
+                            let root = unit_location.ancestors().last().expect("Should exist?");
```

Comment:
can you use context here instead? It's part of anyhow, and I think we should be able to return an anyhow result at this stage. If it's too much work we can skip it and do that later.

Reply:
https://docs.rs/anyhow/latest/anyhow/struct.Error.html#method.context